### PR TITLE
parameter for rules disabling

### PR DESCRIPTION
**Pull request by newvar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,6 +70,16 @@ Of course these two items could be combined into the following:
 
 > let g:EditorConfig_exclude_patterns = ['fugitive://.\*', 'scp://.\*']
 
+#### Disable rules
+
+You could want to override some project-specific EditorConfig rules in global
```
Comment (xuhdev) [inline README.md]: **could** ---> **might**

File: README.md [hunk 2]
```diff
@@ -70,6 +70,16 @@ Of course these two items could be combined into the following:
...
+or local vimrc in some cases, ie to resolve coflicts of trailing whitespace
```
Comment (xuhdev) [inline README.md]: **ie** ---> **e.g.,**

File: plugin/editorconfig.vim [hunk 3]
```diff
@@ -503,18 +507,20 @@ function! s:ApplyConfig(config) " {{{1
 
 " Set the indentation style according to the config values
 
-    if has_key(a:config, "indent_style")
+    if index(g:EditorConfig_disable_rules, 'indent_style') < 0 &&
+                \ has_key(a:config, "indent_style")
```
Comment (xuhdev) [inline plugin/editorconfig.vim]: Instead of repeating this "&&" so many times, I think it's probably better to write a function to do this job.

Comment (newvar) [inline plugin/editorconfig.vim]: sorry, i'm really newbie in vimscript (and english), i didn't get which form of function do you suppose. do you want me to add smth like
```
function isRuleActive(name)
  return index(g:EditorConfig_disable_rules, a:name) < 0 &&
                 \ has_key(a:config, a:name) 
endfunction
```
?

Comment (newvar) [inline plugin/editorconfig.vim]: or just 
```
function isRuleEnabled(name)
  return index(g:EditorConfig_disable_rules, a:name) < 0
endfunction
```
?

Comment (xuhdev) [inline plugin/editorconfig.vim]: The first one would be better. In addition, it is better to have the function be `s:IsRuleActive`, not `isRuleActive".

File: README.md [hunk 4]
```diff
@@ -70,6 +70,16 @@ Of course these two items could be combined into the following:
...
 
+#### Disable rules
+
+You might want to override some project-specific EditorConfig rules in global
+or local vimrc in some cases, e.g., to resolve coflicts of trailing whitespace 
+trimming and buffer autosaving.
+
+> let g:EditorConfig_disable_rules = ['trim_trailing_whitespace']
+
+You are able to disable all of supported EditorConfig properties.
```
Comment (xuhdev) [inline README.md]: One more point: **all of** --> **any**.

**Comment by newvar** [comment]
done

**Comment by newvar** [comment]
oh, sorry, i used to gitignore them)
removed trash from index, sqashed commits

**Comment by xuhdev** [comment]
Thanks!

**Pull request merged**